src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -38,6 +47,9 @@ public static final long divisionUsingRecursion(int a, int b) {
     }
 
     public static final long divisionUsingMultiplication(int a, int b) {
+        if (b == 0) {
+            throw new IllegalArgumentException("Cannot divide by zero");
+        }
```

Comment:
Checks if b is equal to 0, then if true, throw an `IllegalArgumentException` in divisionUsingMultiplication function